Docs: basic user guide (#30)
numba/pixie: docs/source/user_guide/install.md

# Setup and Installation

To install with conda:

```bash
% conda install -c numba -c numba/label/dev pixie 
```

(Note: `numba/label/dev` required for the dev version of `llvmlite`)

> To work with C source code, including Cython, you need clang=14
> 
> ```bash
> % conda install clang=14
> ```

Reviewer: ````suggestion
To work with C source code, you need clang=14, and to work with cython you'll also need cython itself.

```bash
% conda install clang=14 cython
```
````
